help/configure-default-view.md

```diff
@@ -5,8 +5,8 @@ to the Zulip web app. You can also navigate to the default view via
 keyboard shortcuts.
 
 The default views available in Zulip are
-[Recent conversations](/help/recent-conversations) and
-[All messages](/help/all-messages). See
+[Recent conversations](/help/recent-conversations),
+[All messages](/help/all-messages), and [**Inbox**](/help/inbox). See
 [Reading strategies](/help/reading-strategies) for recommendations
```

Comment:
Let's make the order Inbox > Recent conversations > All messages, here and throughout. I've pinged @amanagr to make the same change in the dropdown for default views as well.